frontend/src/scenes/session-recordings/filters/SimpleSessionRecordingsFilters.tsx

```diff
@@ -1,116 +1,109 @@
-                        type={PropertyFilterType.Person}
-                        propertyKey={key}
-                        label={label}
-                        disabled={personProperties.some((property) => property.key === key)}
-                        onChange={(newProperties) => {
-                            const properties = filters.properties || []
-                            setFilters({ ...filters, properties: [...properties, ...newProperties] })
-                        }}
-                    />
-                ))}
-                <SimpleSessionRecordingsFiltersInserter
-                    type={PropertyFilterType.Event}
-                    propertyKey="$current_url"
-                    label="URL"
-                    disabled={!!pageviewEvent}
-                    onChange={(properties) => {
-                        const events = filters.events || []
-                        setLocalFilters({
-                            ...filters,
-                            events: [
-                                ...events,
-                                {
-                                    id: '$pageview',
-                                    name: '$pageview',
-                                    type: EntityTypes.EVENTS,
-                                    properties: properties,
-                                },
-                            ],
-                        })
-                    }}
-                />
-                {displayNameProperties.length === 0 && (
-                    <Link to={urls.settings('project', 'person-display-name')}>
-                        <LemonButton size="small" className="whitespace-nowrap" icon={<IconPlus />}>
-                            Add person properties
-                        </LemonButton>
-                    </Link>
-                )}
-            </div>
+    const onClickPersonProperty = (key: string): void => {
+        setFilters({
+            ...filters,
+            properties: [
+                ...personProperties,
+                { type: PropertyFilterType.Person, key: key, value: null, operator: PropertyOperator.Exact },
+            ],
+        })
+    }
+
+    const onClickCurrentUrl = (): void => {
+        const events = filters.events || []
+        setFilters({
+            ...filters,
+            events: [
+                ...events,
+                {
+                    id: '$pageview',
+                    name: '$pageview',
+                    type: EntityTypes.EVENTS,
+                    properties: [
+                        {
+                            type: PropertyFilterType.Event,
+                            key: '$current_url',
+                            value: null,
+                            operator: PropertyOperator.Exact,
+                        },
+                    ],
+                },
+            ],
+        })
+    }
+
+    const items = useMemo(() => {
+        const keys = personProperties.map((p) => p.key)
```

Comment:
<img width="759" alt="Screenshot 2024-02-27 at 10 10 30" src="https://github.com/PostHog/posthog/assets/984817/2d150ee5-5ec6-423d-8744-5d832ffd95f3">

should this be a person country or event country 🤔 

also would be cool if the text input auto-focussed but very much a nice-to-have

Reply:
This should be auto focusing based on https://github.com/PostHog/posthog/blob/8ce6a3706550004ccb346abf592078ab0fbd1f39/frontend/src/lib/components/PropertyFilters/components/OperatorValueSelect.tsx#L159-L160

Think it's a little buggy but going to leave it as is in the hopes that Product Analytics improve these components in future